omniduct/_version.py

```diff
@@ -1,11 +1,13 @@
 import os
 import sys
 
-__all__ = ['__author__', '__author_email__', '__version__']
+__all__ = ['__author__', '__author_email__', '__version__', '__logo__', '__docs_url__']
 
 __author__ = "Matthew Wardrop, Dan Frank"
-__author_email__ = "matthew.wardrop@airbnb.com, dan.frank@airbnb.com"
+__author_email__ = "mpwardrop@gmail.com, dan.frank@airbnb.com"
```

Comment:
```suggestion
__author_email__ = "mpwardrop@gmail.com, danfrankj@gmail.com"
```

Reply:
trying out this new github feature ...

Reply:
Oh... nice. I didn't know you could do that. Looks like that's a new feature.